src/layers/microscopy-viewer-layer/data-utils/tiff-utils.js

```diff
@@ -34,7 +35,8 @@ export function getTiffConnections({ sourceChannels, maxZoom }) {
   const tiffConnections = Object.keys(sourceChannels).map(async channel => {
     const tiff = await fromUrl(sourceChannels[channel]);
     const imageObj = {};
-    for (let i = 0; i < -maxZoom; i++) {
+    for (let i = 0; i < -maxZoom; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
```

Comment:
https://eslint.org/docs/rules/no-await-in-loop makes it sound like we're missing opportunities for parallelization with `await` in a loop. Is that not a concern for us? If not, add a comment explaining why.

Reply:
That makes sense.  That's how the data-fetching functions are implemented anyway so it'd be good to align.